### PR TITLE
update sha256sum for falco. Release was changed to be conda compatible

### PR DESCRIPTION
**Pull request by guilhermesena1**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by guilhermesena1** [comment]
@BiocondaBot please add label

**Comment by guilhermesena1** [comment]
just a comment for the reviewers: My apologies for sending two PRs in two days. I tested installing the program I'm uploading through conda and realized that, because it depended on some external files, some changes needed to be made so it could run just with the binary. I hope it isn't too much of a problem to just merge the file hash on the metadata. Thank you!

File: recipes/falco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: falco
+  version: {{ version }}
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/falco/meta.yaml]: please start with 0

File: recipes/falco/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#to fix problems with htslib
+export C_INCLUDE_PATH=$C_INCLUDE_PATH:${PREFIX}/include
+export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:${PREFIX}/include
+export LIBRARY_PATH=$LIBRARY_PATH:${PREFIX}/lib
+export LD_LIBRARY_PATH=$LIBRARY_PATH:${PREFIX}/lib
+
+#!/bin/bash
```
Comment (bgruening) [inline recipes/falco/build.sh]: the shebang should be the first line

File: recipes/falco/meta.yaml (hunk 1 above)
Comment (guilhermesena1) [reply recipes/falco/meta.yaml]: Hi @bgruening

I had a pr accepted yesterday that started with 0 so this is the next build. My first commit failed the tests stating I had to increase the number. What should I do for the lint test to pass with build 0?

Comment (dpryan79) [reply recipes/falco/meta.yaml]: Ah, from the PR it looks like falco is new, which was the cause of the confusion. You might want to pull the master branch from bioconda next time before making changes :)

Comment (guilhermesena1) [reply recipes/falco/meta.yaml]: understood, and sorry again! Still getting the hang of making conda builds. Thank you so much for accepting the PR so quickly!

**Pull request merged**